Detectors/ITSMFT/ITS/tracking/include/ITStracking/VertexerTraits.h

```diff
@@ -0,0 +1,93 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+///
+/// \file VertexerTraits.h
+/// \brief
+///
+
+#ifndef VERTEXERTRAITS_H_
+#define VERTEXERTRAITS_H_
```

Comment:
same comment here. Please follow some `O2_ITS_...` pattern.

Reply:
done here as well, pattern is `O2_ITS_TRACKING_<MY_CLASSES_HEADERS>_H_`